remappings.txt

```diff
@@ -1,2 +1 @@
-"ds-test/="../lib/ds-test/src/
```

Comment:
this remapping is still needed in order for the hardhat compilation to work.

Reply:
ah makes sense, closing then